### ISSUE TITLE
Readme missing examples

### ISSUE THREAD

**Issue by UnitedMarsupials-zz** [BartMassey/parsecvs#5]
How would I use this tool? It compiled nicely, but, when I run it:
`find RCS -name '*,v' | xargs /tmp/parsecvs/parsecvs -A map`
I just get some text dumped to stdout. There seems to be some system to it, but what do I do with it? Feed it to some other tool (like `git` with some special flags)?

This is the information, that README files are usually for. What's currently in it may belong to something like HISTORY, but README show concentrate on usage instructions and other information, the would be user _ought_ to read...

There is something like a man-page in `parsecvs.asc`, but that has no real examples either. And the sole example offered in it elicits an error:
```
parsecvs: invalid option -- k
Try `/tmp/parsecvs/parsecvs --help' for more information.
```

**Comment by BartMassey** [comment]
Yeah, parsecvs is not well-documented. As it currently stands, its output is currently intended for `git fast-import`, so you can take a look at the manpage for that. I have no idea why `-k` no longer works.

Honestly, I haven't even thought about this code for years, so I don't have much to offer. Patches welcome as always.
